packages/bpk-tokens/src/web/base/colors.json

```diff
@@ -5,6 +5,78 @@
     "category": "colors"
   },
   "props": {
+    "LINE_LIGHT_COLOR": {
+      "value": "{!SKY_GRAY_TINT_05}"
+    },
+    "LINE_DARK_COLOR": {
+      "value": "{!BLACK_TINT_04}"
+    },
```

Comment:
Should we expose dark mode colours? We don't support DM on web so in theory no one should be using them 🤔

Reply:
Isn't that though how our semantic colours are created? It takes both a light and dark colour to generate it or have I got that wrong?

https://backpack.github.io/guidelines/colour/#dark-mode-dynamic-colours

Reply:
Semantic just means that the alias has a specific meaning. eg `backgroundColor`, `dangerColor`, `successColor`, `calendarLowPriceColor`.
The semantic colours for app are mostly dynamic (light and dark pairs) because the apps support light and dark mode, but on web where we only support light the semantic colours should probably just have one variant